src/lib/nexus-schema-stateful/stateful-nexus-schema.ts

```diff
@@ -75,7 +74,7 @@ export interface NexusSchemaStatefulBuilders {
 }
 
 type NexusSchemaTypeDef =
-  | AllTypeDefs
+  | NexusSchema.core.AllTypeDefs
```

Comment:
I thought somehow this stuff has caused issue.

I'll leave it in this PR as anyways it was a refactor improvement.